lanterndb/lantern.ipynb

```diff
@@ -44,7 +44,7 @@
     "api_url = \"https://api.ecliptor.com/ingest/pdf\"\n",
     "# Request payload\n",
     "payload = {\n",
-    "    \"url\": pdf_url\n",
+    "    \"url\": api_url\n",
```

Comment:
This is actually gonna be the pdf_url - the post request is to the API endpoint but what we need is a URL that leads to the pdf